### DISCUSSION TITLE
Get Started

### DISCUSSION THREAD

**Discussion by tyohan** [inlivedev/inlive-js-sdk#47] [Announcements]
Hi developers,

Welcome to inLive Github discussion. We're using this Github discussion to get your questions, feedback, ideas, or anything related with inLive services. 

Inlive is still in the beta phase, and we have many ideas about what we want to build next. But we want to hear more from you to decide what our next steps are from here. 

For private support, feel free to reach out through [our email ](mailto:support@inlive.app).

**Comment by atdevten** [comment]
Hi, I'm Ahn. 
How to contribute to the project. 
I have exp build video call with pion, not too much but I want to research and deep dive in webrtc.

**Reply by tyohan** [reply]
Hi Ahn @ATtendev,

Since we just released [our open-source SFU library](https://github.com/inlivedev/sfu), I assumed your interest is related to that library.

Because we currently focus on integrating the library into our product, which is still in a very early development phase, this library currently focuses on what our product needs, and external direct contribution will be less prioritized for now.

The easiest way to contribute is by trying to develop your own WebRTC app project using this library. With that, you can help us test this library in real-world scenarios and provide more feedback. Using this library, you can learn more about the basic things about SFU like broadcasting media tracks, perfect negotiation, etc.

I will also share more information and knowledge on the repo to help you understand the project and how we design [our next feature](https://github.com/inlivedev/sfu#features). Please feel free to ask and talk about that feature if you have a specific interest.

Let me know if you have any other ideas on contributing.
